GitCandy/Ssh/Services/ConnectionService.cs

```diff
@@ -75,7 +90,11 @@ private void HandleMessage(ChannelRequestMessage message)
                         {
                             RecipientChannel = FindChannelByServerId<Channel>(message.RecipientChannel).ClientChannelId
                         });
-                    throw new SshConnectionException(string.Format("Unknown request type: {0}.", message.RequestType));
+
+                    if (!IsRequestTypeTolerated(message))
```

Comment:
Thanks for your coding and sorry for deferred reply!
Here, could you please print `message.RawBytes` to show what the package actually is. To help me understanding the mechanism.